docs/Home.md

```diff
@@ -0,0 +1,87 @@
+ 3. Enter a name for your cloud configuration and the Project ID that you will be using to deploy the instances.
+ 4. In the Service Account Credentials dropdown, select the credentials that you uploaded earlier.
+
+### Instance configurations
+An instance configuration allows you to map an instance to a set of labels that Jenkins can use to determine when a job requires a particular type of instance. You can create many instance configurations per GCE configuration.
+
+1. Click on the Add button.
+2. In the Name Prefix field, choose a prefix for the name of the instances that will be deployed.
+3. Set a Description that identifies what this instance configuration will be used for.
+4. Optionally, set a label in the Label field that will allow you to restrict jobs to only run on this type of node.
+5. Select a Region and Zone to define where instances will be launched.
+6. Select the Machine Type for this instance configuration which defines the number of cores and RAM that will be allocated.
+7. Select the Network and Subnetwork that the instance will be deployed into.
+8. For the Boot Disk configuration choose an image that has Java 8 installed and on its default path.
+
+Once complete you should be able to create jobs that restrict their builds to the label you selected. Instances will provision on demand. When no builds have been run for the configured Node Retention Time (default 6 minutes) the instances will be terminated.
+
+### Advanced configurations
+Instance configurations have many options that were not listed above. A few of the important ones are explained below.
+
+* Preemptible - instances provisioned by Jenkins will be launched as Preemptible VMs these are up to 80% less expensive than normal VMs but can be terminated at any time. When using this setting, ensure that builds can be retried without impacting your workload.
+* Disk Type and Size - dictates the performance of the filesystem that your agents are running on. Note that in GCE, larger disks get higher IOPS and throughput.
+* Network tags - these tags will be applied to the instances provisioned by Jenkins. These should be set to allow the Jenkins master to access port 22 on the Jenkins agents. More info on firewall rules in GCE is available here.
+* External IP - dictates whether the instance should receive an external routable IP address. In GCE, you will need to have either an external IP or a NAT gateway setup in order to download anything from the internet.
+* Startup Script - defines a set of commands that should be run before making the instance available for running your jobs. For more info, review the startup script docs.
+* GPUs - attach 1 or more GPUs to the instance. For more info, visit the GCE GPU docs.
+* Service Account E-mail - sets the service account that the instance will be able to access from metadata. For more info, review the service account documentation.
+
+### Feature requests and bug reports
```

Comment:
I think this section should go in the readme rather than these docs.

Reply:
Yea, it's already in the README. Good point. I've removed it since it'll just be redundant.